tbx/work/models.py

```diff
@@ -268,6 +272,21 @@ def read_time(self):
         else:
             return "x"
 
+    @property
+    def listing_image(self):
```

Comment:
:+1:

Reply:
With the addition of this property on both Work Pages and Historical Work Pages, then it would be a good idea to update this accordingly

https://github.com/torchbox/torchbox.com/blob/691e7cb3b35092637cd3325aee46b002be3daad5/tbx/work/models.py#L353-L354

So that we now have

```python
"listing_image": work.listing_image,
```

Reply:
Updated in 9c96331240565cc7ba5c9ce103467f6a9311bd9f